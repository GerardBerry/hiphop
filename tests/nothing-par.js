var rjs = require("../xml-compiler.js");

var prg = <rjs.reactivemachine name="nothingpar">
  <rjs.parallel>
    <rjs.sequence>
      <rjs.nothing/>
      <rjs.pause/>
    </rjs.sequence>
    <rjs.nothing/>
  </rjs.parallel>
</rjs.reactivemachine>;

exports.prg = prg;
