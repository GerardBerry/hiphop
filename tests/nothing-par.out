nothingpar> ;
--- Output:
nothingpar> ;
--- Output:
nothingpar> ;
--- Output:
nothingpar> !reset;
--- Automaton nothingpar reset
nothingpar> ;
--- Output:
nothingpar> ;
--- Output:
